<launch>
    <!-- X Supplies -->
    <node name="PSU4" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="PSU_COM" type="str" value="/dev/ttyUSB4"/>
        <param name="vConv" type="double" value="0.01"/>
        <param name="debugMode" type="bool" value="false"/> <!-- debugMode="true" -->
    </node>
    <node name="PSU5" pkg="ros_coils" type="PSU_node" output="screen">
        <param name="PSU_COM" type="str" value="/dev/ttyUSB5"/>
        <param name="vConv" type="double" value="0.01"/>
        <param name="debugMode" type="bool" value="false"/> <!-- debugMode="true" -->
    </node>

    <!-- todo: replace this with a debug node for Y polarity. -->
    <node name ="yPolarityDebug" pkg="ros_coils" type="yPolarityDebug" output="screen">
        <param name="yRoot" type="str" value="/PSU4"/>
    </node>

</launch>